draft-ietf-rtcweb-jsep.xml

```diff
@@ -444,6 +444,12 @@ setRemote(ANSWER) |                                   |
           sections when RtpTransceivers are created but not yet
           associated with a m= section, or if RtpTransceivers have been
           stopped and disassociated from m= sections. An RtpTransceiver
+          is said to be associated with an m= section if and only if
+          its mid attribute is equal to the MID of that m= section, or a
+          mapping exists between the transceiver and the index of that m=
+          section. The index-based mapping is required for remote
```

Comment:
I think this is formally correct, but if and only if <blah> or <blah> is hard.

Maybe "An RtpTransceiver is said to be associated with an m= section if <blah> or <blah>. Otherwise the transceiver is said to be "disassociated"